### PR TITLE
Add fty-service-status

### PR DESCRIPTION
**Pull request by perrettecl**

### CODE REVIEW

File: src/fty_security_wallet.cc [hunk 1]
```diff
@@ -34,15 +34,23 @@
...
+#include <fty_service_status.h>
 
 //functions
 void usage();
 
 int main (int argc, char *argv [])
 {
+    //start service status
+    fty::ServiceStatusPluginWrapperCollection statusProviders(SECURITY_WALLET_AGENT);
+    int pluginLoaded = statusProviders.addAll("/usr/libexec/service_status_plugins/", std::regex(".*\\.so"));
```
Comment (aquette) [inline src/fty_security_wallet.cc:46]: this path should be a config point, managed by autotools and configure call

**Review by aquette — CHANGES_REQUESTED** [review]
there is a conflict, a need for improvement (plugin path hard coded) and a very fast review led me to being astonished by some of the gsl related generation changes (sure you're running the latest version). But for these last, I rely on @jimklimov to check and elaborate if needed

File: src/fty_security_wallet.cc (hunk 1 above)
Comment (FrancoisRegisDegott-eaton) [reply src/fty_security_wallet.cc:46]: target the master branch?

**Comment by FrancoisRegisDegott-eaton** [comment]
obsolete (closed)

**Pull request closed without merging**